packages/pds/src/config/env.ts

```diff
@@ -119,6 +119,9 @@ export const readEnv = (): ServerEnvironment => {
 
     // fetch
     fetchDisableSsrfProtection: envBool('PDS_DISABLE_SSRF_PROTECTION'),
+
+    // Repo revision cache
+    repoRevCacheMaxAge: envInt('PDS_REPO_REV_CACHE_MAX_AGE'), // Seconds (use 0 to disable)
```

Comment:
To make the PDS config more uniform I recommend we switch this to be `_MAX_TTL` and to accept a value in milliseconds.